build.gradle

```diff
@@ -44,7 +44,7 @@ allprojects {
             url = 'https://repo.maven.apache.org/maven2'
         }
         maven {
-            url = 'https://maven.pkg.github.com/ballerina-platform/ballerina-lang'
+            url = 'https://maven.pkg.github.com/ballerina-platform/*'
```

Comment:
We can leave this as it is since its only one repository

Reply:
Fixed.